README.md

```diff
@@ -31,22 +31,67 @@ Once installed, the following ``py.test`` command-line parameters are available:
 
 ```bash
 py.test \
-    [--ansible-inventory <path_to_inventory>] \
-    [--ansible-host-pattern <host-pattern>] \
-    [--ansible-connection <plugin>] \
-    [--ansible-module-path <path_to_modules] \
-    [--ansible-user <username>] \
-    [--ansible-sudo] \
-    [--ansible-sudo-user <username>]
+    [--inventory <path_to_inventory>] \
+    [--host-pattern <host-pattern>] \
+    [--connection <plugin>] \
+    [--module-path <path_to_modules] \
+    [--user <username>] \
+    [--become] \
+    [--become-user <username>] \
+    [--become-method <method>] \
+    [--limit <limit>] \
+    [--check]
 ```
 
 The following fixtures are available:
 
 
+### Fixture ``ansible_adhoc``
+
+The `ansible_adhoc` fixture returns a function used to initialize a `HostManager` object.
+
+```python
+def test_all_the_pings(ansible_adhoc):
+    result = ansible_adhoc().all.ping()
+    assert result.is_successful
+
```

Comment:
Thanks, I went back and forth on exposing an `is_successful`, but chose not to as it's not surfaced by ansible itself. There is no "success" status back from a task (see https://github.com/ansible/ansible/blob/fceb71128eecf543dfa779d923c4e15d9a28e366/lib/ansible/executor/task_result.py#L50-L73).  I'll update the README.md accordingly.  Good catch!